src/material/stepper/stepper-animations.ts

```diff
@@ -25,9 +25,9 @@ export const matStepperAnimations: {
   /** Animation that transitions the step along the X axis in a horizontal stepper. */
   horizontalStepTransition: trigger('stepTransition', [
     state('previous', style({transform: 'translate3d(-100%, 0, 0)', visibility: 'hidden'})),
-    // Transition to '', rather than `visible`, because visibility on a child element overrides
+    // Transition to `unset`, rather than `visible`, because visibility on a child element overrides
     // the one from the parent, making this element focusable inside of a `hidden` element.
-    state('current', style({transform: 'none', visibility: ''})),
+    state('current', style({transform: 'none', visibility: 'unset'})),
```

Comment:
[According to MDN](https://developer.mozilla.org/en-US/docs/Web/CSS/unset), `unset` isn't supported on IE.

Reply:
@crisbeto You are right.
According to the logic of animation, if we use visible on child element, this overrides visibility which is set by the parent.
To overcome this issue visibility was set to empty string, which leads to the error I have mentioned in the screen shot.
If we want to preserve the parent visibility then `unset` is the best option we have.
Except IE `unset` will on every browsers and we also not face "Failed to execute 'animate' on 'Element' : Partial keyframes are not supported" error.
If you have better solution please let me know.

Reply:
For what it's worth, I can't reproduce the error locally against IE11.

Reply:
This error is can be reproduced locally in Chrome 77 & Firefox 56.

Reply:
@mmalerba we can use `inherit`. Should I update the code with `inherit`?

Reply:
We only support the last 2 major versions of Chrome and Firefox.

Reply:
@crisbeto There are many products who are using angular material and they can not say their customer to update browser. This simple fix will solve the problem and this will put more faith in angular material.

Reply:
In the animation code we are setting invalid property value. That is the main reason of error. If we keep invalid property value, it is possible that in future version of browser this error may occur.

Reply:
just another point of view... if a browser is embedded in other applications, they get rarely updates. For example electron-apps, all kinds of Smart-TVs, Kiosksystems (https://www.provisio.com/web/us/products/windows-kiosk-software-sitekiosk), and many more

Reply:
I'm not arguing against fixing the issue, but we shouldn't be replacing one invalid value with another. Also I'm not sure why I don't see the error against my local IE11.

Reply:
@crisbeto `inherit` will work in IE 11 also. Let me update this pr.

Reply:
Pr is updated with `inherit` instead of `unset`.